tests/lagrangian_detectors_increasing_rotation/analytical_traj.py

```diff
@@ -0,0 +1,24 @@
+from pylab import *
```

Comment:
Add a comment saying what this script does.  Appears to both generate Xvals and Yvals and open a plot.

Reply:
Done below